File: app/src/main/java/art/luaj/hyperisle/ui/fragment/mainnav/home/HomeFragment.java
package art.luaj.hyperisle.ui.fragment.mainnav.home;

import android.view.LayoutInflater;
import android.view.ViewGroup;

import art.luaj.hyperisle.databinding.FragmentHomeBinding;
import art.luaj.hyperisle.ext.BaseFragment;

public class HomeFragment extends BaseFragment<FragmentHomeBinding> {
    @Override
    protected FragmentHomeBinding inflate(LayoutInflater inflater, ViewGroup container, boolean attachToRoot) {
        return FragmentHomeBinding.inflate(inflater, container, attachToRoot);
    }

    @Override
    protected void initFragment(FragmentHomeBinding binding) {

    }
}

File: app/src/main/java/art/luaj/hyperisle/ui/fragment/nav/HomeFragment.kt
package art.luaj.hyperisle.ui.fragment.nav

import androidx.lifecycle.ViewModel
import androidx.navigation.findNavController
import art.luaj.hyperisle.R
import art.luaj.hyperisle.databinding.FragmentHomeBinding
import art.luaj.hyperisle.ui.base.BaseFragment

class HomeFragment : BaseFragment<FragmentHomeBinding, ViewModel>(
    FragmentHomeBinding::inflate,
    null
) {
    override fun initFragment(binding: FragmentHomeBinding, viewModel: ViewModel?) {
        /*
        binding.button.setOnClickListener {
            requireActivity().findNavController(R.id.mainView)
                .navigate(R.id.action_mainNavFragment_to_startFragment)
        }

         */
    }
}